eduroam.md

```diff
@@ -11,36 +11,53 @@ Starting after 2022 commencement RIT is replacing the 'RIT' wifi network with a
 
-## Required Programs
+## Requirements
 
 ### GUI Setup
 - NetworkManager and a desktop environment that uses it (KDE, GNOME, LXDE, Cinnamon, MATE)
 - Terminal emulator
 - Root/`sudo` permissions
 
 ### CLI Setup
-- `wpa_supplicant`, `iwd`, or `nmcli`
+- A terminal network configuration tool and an RIT-distributed personal certificate (see "Initial Setup")
+    - For `wpa_supplicant`and `nmcli`, choose the `P12` option
+    - For `iwd`, choose the `PEM` option
+    - Other network managers are not currently documented
 - Terminal emulator
 - Root/`sudo` permissions
-- Some text editor (terminal text editors such as `micro`, `nvim`/`vim`, or `emacs` are easiest for this purpose, but not strictly necessary)
+- Some text editor (terminal text editors [`nano`, `micro`, `nvim`/`vim`, `emacs`, etc] are easiest for this purpose, but not strictly necessary)
 
 ## Initial Setup
 
 Initial setup requires internet access to download certificates. This can be done on campus by connecting to the `RIT-WiFi` network.
 
-1. Go to [*https://rit.edu/wifi*](https://rit.edu/wifi), then select `eduroam`. ![](/assets/img/eduroam/wifi-page.png)
+1. Go to [*https://rit.edu/wifi*](https://rit.edu/wifi), then select `eduroam`. 
 
-2. In the 'Select your device' dropdown, select the `user-defined` option. ![](/assets/img/eduroam/select-os.png)
+![](/assets/img/eduroam/wifi-page.png)
 
-3. Click 'Sign in' and enter your RIT credentials when prompted. ![](/assets/img/eduroam/start-user-cert.png)
+2. In the 'Select your device' dropdown, select the `user-defined` option. 
 
-4. Once you are logged in, you will be asked to create a certificate. Enter a name you would like to give this certificate in the 'User Description' input box, then, click the 'Create' button. ![](/assets/img/eduroam/create-user-cert.png)
+![](/assets/img/eduroam/select-os.png)
 
-5. You will then be prompted to create a password to protect your private key. This will be needed to set up eduroam on a new device. If you are unsure, you can use [1Password's public generator](https://1password.com/password-generator/?) which has a "memorable password" mode. The password you choose must be at least six (but no more than 16) characters long and contain a letter, number, and symbol. ![](/assets/img/eduroam/password.png)
+3. Click 'Sign in' and enter your RIT credentials when prompted. 
 
-6. Choose a format to download your certificate in. If you aren't sure what you need, select P12 format and save the file somewhere you can get to it easily. If you are using `iwd` and the terminal-only setup, select the PEM option. ![](/assets/img/eduroam/cert-download.png)
+![](/assets/img/eduroam/start-user-cert.png)
 
```

Comment:
i see what you did here with trying to put the images on their own lines, but this also broke the numbering for some reason.

After looking into this it appears to have been an issue with the built in markdown parser kramdown. switching to commonmark has fixed it